site/gatsby-site/cypress/e2e/integration/submit.cy.js

```diff
@@ -1324,4 +1324,65 @@ describe('The Submit form', () => {
 
     cy.contains('Please review. Some data is missing.').should('not.exist');
   });
+
+  it('Should allow two submissions in a row', () => {
+    cy.visit(url);
+
+    cy.intercept('GET', parserURL, parseNews).as('parseNews');
+
+    cy.conditionalIntercept(
+      '**/graphql',
+      (req) => req.body.operationName == 'InsertSubmission',
+      'insertSubmission',
+      {
+        data: {
+          insertOneSubmission: { __typename: 'Submission', _id: '6272f2218933c7a9b512e13b' },
+        },
+      }
+    );
+
+    cy.get('input[name="url"]').type(
+      `https://www.arstechnica.com/gadgets/2017/11/youtube-to-crack-down-on-inappropriate-content-masked-as-kids-cartoons/`
+    );
+
+    cy.get('[data-cy="fetch-info"]').click();
+
+    cy.wait('@parseNews');
+
+    cy.get('input[name="authors"]').type('Something');
+
+    cy.setEditorText(
```

Comment:
The fetch info button fills the text field, you shouldn't need to set a text manually 🤔

Reply:
You're right... Although it was like this before on other tests, but I tested and it works without the manual set. So maybe this wasn't working right on a previous version of Cypress. I removed the manual setEditorText when using the fetch Info option